account/src/main/java/org/killbill/billing/account/dao/DefaultAccountDao.java

```diff
@@ -334,4 +334,15 @@ public Integer inTransaction(final EntitySqlDaoWrapperFactory entitySqlDaoWrappe
             }
         });
     }
+
+    @Override
+    public List<AuditLogWithHistory> getEmailAuditLogsWithHistoryForId(final UUID accountEmailId, final AuditLevel auditLevel, final InternalTenantContext context) throws AccountApiException {
+        return transactionalSqlDao.execute(true, AccountApiException.class, new EntitySqlDaoTransactionWrapper<List<AuditLogWithHistory>>() {
```

Comment:
Can you use the other signature (without the exception)? It doesn't look like `AccountApiException` is thrown here (same remark below).